modules/nf-core/nextdenovo/meta.yml

```diff
@@ -0,0 +1,70 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: "nextdenovo"
+description: string graph-based de novo assembler for long reads (CLR, HiFi and ONT)
+keywords:
+  - assembler
+  - long reads
```

Comment:
Hmm, does the linter allow two words?

Reply:
It appear it does. Still, I added a dash to be sure